### PR TITLE
doc / digifinex docs

### PR DESCRIPTION
**Pull request by JeremyKono**
Digifinex documentation

### CODE REVIEW

File: content/exchange-connectors/digifinex.mdx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+### Creating Digifinex API Keys
+
+1. Log in to your account at https://www.digifinex.com/en-ww/login, then click your profile > User Center (If you do not have an account, you will have to create one and verify your ID.)
+
+<Callout
+  type="tip"
+  body="You must enable 2FA in your Digifinex account to create the API key. [How to enable 2FA]?"
+  link={[
+    "https://www.google.com/search?q=how+to+enable+2FA+on+digifinex&oq=how+to+enabl&aqs=chrome.0.69i59j69i57j69i59l2j69i61l3.1434j0j4&sourceid=chrome&ie=UTF-8",
```
Comment (CrimsonJacket) [inline content/exchange-connectors/digifinex.mdx:46]: Do they not have a support/help ticket for this? We should include a Google search link here

File: content/exchange-connectors/digifinex.mdx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+  body="If you lose your Secret Key, you can delete the API and create a new one. However, it will be impossible to reuse the same API."
+/>
+
+## Miscellaneous Info
+
+<Callout
+  type="note"
+  bullets={[
+    "Websocket error / timeout may occur every 2 hours",
+    "Low rate limit, I think we need to add a link to their trade rule so that users should be aware on this limit and avoid getting ban look at link [here]",
```
Comment (CrimsonJacket) [inline content/exchange-connectors/digifinex.mdx:77]: The tone here doesn't seem right. It's should be more instructional

File: content/exchange-connectors/digifinex.mdx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+### Minimum order sizes
+
+Minimum order size varies per market. All minimum trade quantities can be found in the following public API:
+
+```
+https://openapi.digifinex.com/v3/markets
+```
+
+`BTC-USDT min 0.00005.`
```
Comment (CrimsonJacket) [inline content/exchange-connectors/digifinex.mdx:90]: Would be good if you can explain to users what this means

**Review by CrimsonJacket — CHANGES_REQUESTED** [review]
Requested some changes. Should be more careful about including the appropriate links and choice of words.

**Comment by JeremyKono** [comment]
Changes are in the most recent commits

**Review by CrimsonJacket — APPROVED** [review]
LGTM

**Pull request merged**